crates/telio-model/src/api_config.rs

```diff
@@ -268,6 +268,13 @@ where
     Ok(Some(eps))
 }
 
+/// Turns on the no link detection mechanism
+#[derive(Default, Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureNoLinkDetection {
+    /// Configurable rtt
+    pub rtt: Option<u64>,
```

Comment:
It's quite confusing, to be honest. Can you use `Duration`?

Reply:
Every duration in feature structures are defined like this and I would prefer to keep it as it is for consistency reasons. I converted it in Duration lower in the lib as you suggested.

Reply:
Oh, yeah. I haven't noticed it's being deserialized. You're right on this one :+1:

Reply:
You should add _ms or unit in field name to make it clear to user.